simple_calculator/index.html

```diff
@@ -39,7 +39,7 @@ <h1>Calculator Project</h1>
 
             <div class="row">                
                 <div class="cal-button">9</div>
-                <div class="cal-button equation">C</div>
+                <div class="cal-button equation reset">C</div>
```

Comment:
그래도 id를 일단 넣어둡시다. 계속 다시 시도해볼 수 있도록이요.

```suggestion
                <div id="reset" class="cal-button equation reset">C</div>
```